app/models/users/notification_setting.rb

```diff
@@ -0,0 +1,16 @@
+module Users
+  class NotificationSetting < ApplicationRecord
+    self.table_name_prefix = "users_"
+
+    validates :user_id, presence: true
```

Comment:
Do we really need this if we use proper foreign keys?

Reply:
No, probably not.

Reply:
So are we removing this line?

Reply:
@msarit yeah we can remove it